ledger/heavy/executor/initial_state_keeper.go

```diff
@@ -73,7 +76,9 @@ func NewInitialStateKeeper(
 		indexStorage:          indexStorage,
 		dropStorage:           dropStorage,
 		syncPulse:             jetKeeper.TopSyncPulse(),
+		jetTree:               jet.NewTree(true),
```

Comment:
`StateKeeper` is a singleton. This field is a temporary object to perform calculations. It might produce serious and hard to find bugs.